diff -urN a/apps/dashboard/app/views/layouts/_footer.html.erb b/apps/dashboard/app/views/layouts/_footer.html.erb
--- a/apps/dashboard/app/views/layouts/_footer.html.erb	2023-02-07 17:08:52.000000000 +0200
+++ b/apps/dashboard/app/views/layouts/_footer.html.erb	2023-03-06 15:10:57.270625209 +0200
@@ -11,5 +11,11 @@
       %>
     <% end %>
   </div>
-  <span>OnDemand version: <%= Configuration.ood_version %></span>
+  <span>
+    Puhti web interface: <b>Release CSC_OOD_RELEASE</b> <br> Based on OnDemand version: <%= Configuration.ood_version %>
+    <br>
+    <a href='/public/?info=cookie_policy' class='text-secondary'>Cookie policy</a>
+    <br>
+    <a href='/public/?info=accessibility_statement' class='text-secondary'>Accessibility statement</a>
+  </span>
 </footer>
