diff -urN a/apps/dashboard/app/views/layouts/_footer.html.erb b/apps/dashboard/app/views/layouts/_footer.html.erb
--- a/apps/dashboard/app/views/layouts/_footer.html.erb	2023-02-07 17:08:52.000000000 +0200
+++ b/apps/dashboard/app/views/layouts/_footer.html.erb	2023-03-06 15:10:57.270625209 +0200
@@ -11,5 +11,11 @@
       %>
     <% end %>
   </div>
-  <span>OnDemand version: <%= Configuration.ood_version %></span>
+  <span>
+    LUMI web interface: <b><%= CSCConfiguration.release_name %></b> <br> Based on OnDemand version: <%= Configuration.ood_version %>
+    <br>
+    <a id='cookie-policy-toggle' class='text-secondary' tabindex='0'>Cookie policy</a>
+    <br>
+    <a id='accessibility-statement-toggle' class='text-secondary' tabindex='0'>Accessibility statement</a>
+  </span>
 </footer>
