diff -urN a/apps/dashboard/app/views/layouts/application.html.erb b/apps/dashboard/app/views/layouts/application.html.erb
--- a/apps/dashboard/app/views/layouts/application.html.erb	2024-02-12 12:31:02.049457483 +0200
+++ b/apps/dashboard/app/views/layouts/application.html.erb	2024-02-12 12:32:59.600963188 +0200
@@ -24,6 +24,11 @@
   <meta name="profile" content="<%= @user_configuration.profile %>">
 
   <%= render partial: 'layouts/config' %>
+  <% begin %>
+    <%= render partial: "widgets/extra_head" %>
+  <% rescue ActionView::MissingTemplate %>
+    <% Rails.logger.error("missing widgets/_extra_head.html.erb") %>
+  <% end %>
 </head>
 <body>
   <header>
