diff -urN a/apps/dashboard/app/views/layouts/application.html.erb b/apps/dashboard/app/views/layouts/application.html.erb
--- a/apps/dashboard/app/views/layouts/application.html.erb	2023-03-06 14:51:02.735020347 +0200
+++ b/apps/dashboard/app/views/layouts/application.html.erb	2023-03-06 14:51:39.190153938 +0200
@@ -22,6 +22,11 @@
     <meta name="turbolinks-root" content="<%= ENV['RAILS_RELATIVE_URL_ROOT'] || "/" %>">
   <% end %>
 <%= render partial: '/layouts/nav/styles', locals: { bg_color: Configuration.brand_bg_color, link_active_color: Configuration.brand_link_active_bg_color } %>
+  <% begin %>
+    <%= render partial: "widgets/extra_head" %>
+  <% rescue ActionView::MissingTemplate %>
+    <%# Rails.logger.error("missing widgets/_extra_head.html.erb") %>
+  <% end %>
 </head>
 <body>
   <header <%= 'data-turbolinks-permanent' if Configuration.turbolinks_enabled? %>>
